<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_Settings">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="icon">
       <iconset theme="clamui">
        <normaloff/>
       </iconset>
      </attribute>
      <attribute name="title">
       <string>ClamUI</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea_ClamUI">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>739</width>
            <height>350</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_General">
             <property name="title">
              <string>Allgemein</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="3" column="0">
                 <widget class="QCheckBox" name="checkBox_MenuBar">
                  <property name="toolTip">
                   <string>Die Menüleiste ein- oder ausblenden.</string>
                  </property>
                  <property name="text">
                   <string>Menü anzeigen.</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBox_Autostart">
                  <property name="text">
                   <string>ClamUI automatisch starten.</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="checkBox_HideWindow">
                  <property name="text">
                   <string>Fenster beim Start nicht anzeigen.</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QCheckBox" name="checkBox_IconOnDesktop">
                  <property name="text">
                   <string>Icon auf dem Desktop anzeigen.</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QCheckBox" name="checkBox_ToolBar">
                  <property name="toolTip">
                   <string>Die Werkzeugleiste ein- oder ausblenden.</string>
                  </property>
                  <property name="text">
                   <string>Werkzeugleiste anzeigen.</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QCheckBox" name="checkBox_StatusBar">
                  <property name="toolTip">
                   <string>Die Statusleiste ein- oder ausblenden.</string>
                  </property>
                  <property name="text">
                   <string>Statusleiste anzeigen.</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_Language">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>S&amp;prache ändern</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>&amp;Sprache wechseln (Benötigt einen Neustart):</string>
                  </property>
                  <property name="buddy">
                   <cstring>comboBox_Language</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_Language"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset>
        <normaloff>../../../../../../../usr/share/doc/packages/clamav/html/clam.png</normaloff>../../../../../../../usr/share/doc/packages/clamav/html/clam.png</iconset>
      </attribute>
      <attribute name="title">
       <string>ClamAV</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_ClamAV">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>739</width>
            <height>350</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Help">
       <property name="text">
        <string>Hilfe</string>
       </property>
       <property name="icon">
        <iconset theme="help-about">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Default">
       <property name="text">
        <string>Standardwerte</string>
       </property>
       <property name="icon">
        <iconset theme="edit-undo">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Save">
       <property name="text">
        <string>Speichern</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok-apply">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Close">
       <property name="text">
        <string>Schließen</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-close">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget_Settings</tabstop>
  <tabstop>scrollArea_ClamUI</tabstop>
  <tabstop>checkBox_Autostart</tabstop>
  <tabstop>checkBox_HideWindow</tabstop>
  <tabstop>checkBox_IconOnDesktop</tabstop>
  <tabstop>checkBox_MenuBar</tabstop>
  <tabstop>checkBox_ToolBar</tabstop>
  <tabstop>checkBox_StatusBar</tabstop>
  <tabstop>groupBox_Language</tabstop>
  <tabstop>comboBox_Language</tabstop>
  <tabstop>pushButton_Help</tabstop>
  <tabstop>pushButton_Default</tabstop>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>pushButton_Close</tabstop>
  <tabstop>scrollArea_ClamAV</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
